app/_includes/events.html

```diff
@@ -7,6 +7,14 @@
 			<h1 class="title feature-header">{{site.data[locale].events.title}}</h1>
 		</div>
 	</div>
+	<div class="row" style="text-align: center">
+		<a href="https://docs.google.com/forms/d/e/1FAIpQLScdvsMHLor8mrDrHa-BQs2XzFb96vIrx3u--LHXWd1g7RncPg/viewform">
+			<div class="btn btn-blue arw">{{site.data[locale].events.register}}
```

Comment:
About the variable: I suggest syncing the naming, since the variable content is the same as well.
Below it is `site.data[locale].host.mapevent_btn`. Here it is `site.data[locale].events.register`.

I suggest renaming both to `site.data[locale].events.register_event_btn` which states more clearly what its about.